### PR TITLE
[enterprise-4.4] BZ1886991 …

### PR DESCRIPTION
**Pull request by jeana-redhat**
Correct information for User-provided certificates for default ingress, restructure modularization of this section to allow xref support.

Cherry Picked from 4448605efdb0e93b01ba37ff05e72aa3c81be91c xref: https://github.com/openshift/openshift-docs/pull/27023

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@ahardin-rh Finally getting back to this in 4.4. 

I have checked against the content in https://docs.openshift.com/container-platform/4.4/security/certificate-types-descriptions.html but would very much appreciate a supplemental peer review since it's been a couple months since I did the 4.5+ version of this CP and also because it's all sort of swimming in front of my eyes... but I am pretty sure I didn't pull any 4.5+ changes into this branch!

Preview is here: http://file.rdu.redhat.com/~jrouth/20210108/enterprise-4.4/security/certificate_types_descriptions/user-provided-certificates-for-api-server.html

File: security/certificate_types_descriptions/service-ca-certificates.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+`service-ca` is an Operator that creates a self-signed CA when an {product-title} cluster is deployed.
+
+== Expiration
+
+A custom expiration term is not supported. The self-signed CA is stored in a secret with qualified name `service-ca/signing-key` in fields `tls.crt` (certificate(s)), `tls.key` (private key), and `ca-bundle.crt` (CA bundle).
+
+Other services can request a service serving certificate by annotating a service resource with `service.beta.openshift.io/serving-cert-secret-name: <secret name>`. In response, the Operator generates a new certificate, as `tls.crt`, and private key, as `tls.key` to the named secret. The certificate is valid for two years.
+
+Other services can request that the CA bundle for the service CA be injected into APIService or ConfigMap resources by annotating with `service.beta.openshift.io/inject-cabundle: true` to support validating certificates generated from the service CA. In response, the Operator writes its current CA bundle to the `CABundle` field of APIService or as `service-ca.crt` to a ConfigMap.
```
Comment (ahardin-rh) [inline security/certificate_types_descriptions/service-ca-certificates.adoc:18]: `APIService` or `ConfigMap` resources

Comment (ahardin-rh) [reply security/certificate_types_descriptions/service-ca-certificates.adoc:18]: to a config map

**Comment by ahardin-rh** [comment]
@jeana-redhat Only some minor style comments from me given the updated guidelines. Looks great!

**Pull request merged**